src/common/src/sys/cgroup.rs

```diff
@@ -0,0 +1,673 @@
+// Copyright 2022 Singularity Data
```

Comment:
Why do we have our own code to interface with cgroups? Can we just use an existing crate so we have less code to maintain?